src/components/input/UIInput.vue

```diff
@@ -117,12 +117,20 @@
             </div>
         </div>
 
-        <UIExpandTransition>
-            <slot v-if="error || $slots.error" name="error">
-                <p class="text-color-error text-sm">
-                    {{ error }}
-                </p>
-            </slot>
+        <UIExpandTransition :height-duration="100" :opacity-duration="200">
```

Comment:
keep it default (as that's consistent with other inputs)

```suggestion
        <UIExpandTransition>
```